niceman/retrace/packagemanagers.py

```diff
@@ -179,6 +271,40 @@ def _create_package(self, pkgname):
         lgr.debug("Found package %s", pkg)
         return pkg
 
+    class MultipleReleaseFileMatch(RuntimeError):
+        pass
+
+    def _find_release_date(self, site, archive):
+        if not site:
+            return None
+        # First find the release file
+        # This uses logic similar to apt.utils.get_release_filename_for_pkg
+        # but restricts matches to the source site
+        rfile = None
+        dirname = apt_pkg.config.find_dir("Dir::State::lists")
```

Comment:
You are correct, apt_pkg needs to be initialized, but that is done with the call to cache = apt.Cache() on line 29:

```
[butch@ubuntu ~] $ python -c 'import apt; import apt_pkg; cache = apt.Cache(); print apt_pkg.config.find_dir("Dir::State::lists")'
/var/lib/apt/lists/
```